diagrams/metametamodel-gen.puml

```diff
@@ -1,7 +1,7 @@
 @startuml
```

Comment:
This file is generated, so I assume it does not need to be reviewed

Reply:
It should actually be compared to the non-generated file in the `organization` repo. Unfortunately, that comparison can't be automated and still needs a human eye because of the “disputed” meta-features that are still in.